Week2/homework/map-filter.js

```diff
@@ -1,14 +1,13 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  const newNumbers = numbers.filter(x => x % 2 === 1).map(y => y * 2);
```

Comment:
In this case, I have no objection against using names such as `x` or `y`, since we are dealing with numbers that in this context have no higher meaning other than being numbers.